docs/source/training.mdx

```diff
@@ -10,385 +10,351 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 
-- remove the columns corresponding to values the model does not expect (here the `"text"` column)
-- rename the column `"label"` to `"labels"` (because the model expect the argument to be named `labels`)
-- set the format of the datasets so they return PyTorch Tensors instead of lists.
+1. Remove the `text` column because the model does not accept the raw text as an input:
 
-Our _tokenized_datasets_ has one method for each of those steps:
+    ```py
+    >>> tokenized_datasets = tokenized_datasets.remove_columns(["text"])
+    ```
 
-```python
-tokenized_datasets = tokenized_datasets.remove_columns(["text"])
-tokenized_datasets = tokenized_datasets.rename_column("label", "labels")
-tokenized_datasets.set_format("torch")
+2. Rename the `label` column to `labels` because the model expects the argument to be named `labels`:
 
-small_train_dataset = tokenized_datasets["train"].shuffle(seed=42).select(range(1000))
-small_eval_dataset = tokenized_datasets["test"].shuffle(seed=42).select(range(1000))
-```
+    ```py
+    >>> tokenized_datasets = tokenized_datasets.rename_column("label", "labels")
+    ```
 
-Now that this is done, we can easily define our dataloaders:
+3. Set the format of the dataset to return PyTorch tensors instead of lists:
 
-```python
-from torch.utils.data import DataLoader
+    ```py
+    >>> tokenized_datasets.set_format("torch")
+    ```
 
-train_dataloader = DataLoader(small_train_dataset, shuffle=True, batch_size=8)
-eval_dataloader = DataLoader(small_eval_dataset, batch_size=8)
+Then you can create a smaller subset of the dataset as previously shown:
+
+```py
+>>> small_train_dataset = tokenized_datasets["train"].shuffle(seed=42).select(range(1000))
+>>> small_eval_dataset = tokenized_datasets["test"].shuffle(seed=42).select(range(1000))
 ```
 
-Next, we define our model:
+### DataLoader
 
-```python
-from transformers import AutoModelForSequenceClassification
+Create a `DataLoader` for your training and test datasets:
 
-model = AutoModelForSequenceClassification.from_pretrained("bert-base-cased", num_labels=2)
+```py
+>>> from torch.utils.data import DataLoader
+
+>>> train_dataloader = DataLoader(small_train_dataset, shuffle=True, batch_size=8)
+>>> eval_dataloader = DataLoader(small_eval_dataset, batch_size=8)
 ```
 
-We are almost ready to write our training loop, the only two things are missing are an optimizer and a learning rate
-scheduler. The default optimizer used by the [`Trainer`] is [`AdamW`]:
+Load your model with the number of expected labels:
 
-```python
-from transformers import AdamW
+```py
+>>> from transformers import AutoModelForSequenceClassification
+model = AutoModelForSequenceClassification.from_pretrained("bert-base-cased", num_labels=5)
+```
+
+### Optimizer and learning rate scheduler
+
+Create an optimizer and learning rate scheduler to fine-tune the model with. Let's use the default optimizer used in [`Trainer`], [`AdamW`]:
+
+```py
+>>> from transformers import AdamW
```

Comment:
I think we should move people out of our version of `AdamW` to use the one in torch.